### PR TITLE
BUG: VCN state must be write-able

### PR DESCRIPTION
**Pull request by n-wbrown**
## Description

The altered field should be writable from the Typhos generated screens.

### CODE REVIEW

**Comment by hhslepicka** [comment]
It seems wrong that users should be able to specify the current state of a device.
Also, this field is being used as the `ERROR` information for the widgets.

**Comment by n-wbrown** [comment]
This isn't so much a interpreted state but a state that we set to determine the control mode for this valve.

File: pcdsdevices/valve.py [hunk 1]
```diff
@@ -272,5 +272,5 @@ class VCN(Device):
                        doc='interlock ok status')
     open_command = Cpt(EpicsSignalWithRBV, ':OPN_SW', kind='normal',
                        doc='Epics command to Open valve')
-    state = Cpt(EpicsSignalRO, ':STATE_RBV', kind='hinted', doc='Valve state')
+    state = Cpt(EpicsSignalWithRBV, ':STATE_RBV', kind='hinted', doc='Valve state')
```
Comment (klauer) [inline pcdsdevices/valve.py]:
```suggestion
    state = Cpt(EpicsSignalWithRBV, ':STATE', kind='hinted', doc='Valve state')
```

**Comment by hhslepicka** [comment]
@n-wbrown in this case the widget is wrong I would say due to the fact that it is not the state of the device based on some feedback but based on what operator sets.
@ghalym would you mind to confirm, please?

File: pcdsdevices/valve.py (hunk 1 above)
Comment (n-wbrown) [reply pcdsdevices/valve.py]: Added. Thanks!

**Comment by ZLLentz** [comment]
pcdsdevices travis is hopelessly broken right now, ignoring

merging with the :+1: review

sad that #389 is being deliberately ignored in a systematic way

**Comment by ZLLentz** [comment]
After lockup I will be enforcing #389 in all PRs unless someone convinces me that the current practice is better

**Pull request merged**